client/src/actions/scoreForm.js

```diff
@@ -0,0 +1,16 @@
+export const Types = {
+	UpdateForm: 'FORM_UPDATE_FORM',
+	ResetForm: 'FORM_RESET_FORM',
+}
+
+export function update() {
```

Comment:
What does this do?

Reply:
That's an action that's used in redux for the form. Right now, it doesn't do anything. Will need it later to send data to the backend.